### PR TITLE
Fix INSERT ... WHERE EXIST

### PR DESCRIPTION
**Pull request by PapillonMcGill**
### Brief summary of changes
select from DUAL table as  we can't use WHERE clause in a SELECT without FROM table

### This resolves issue...
Resolves https://github.com/aces/Loris/issues/4595

### To test this change...

run the SQL/New_patches/2019-05-03-deprecate_useprojects.sql script on a version 20 database

### CODE REVIEW

File: SQL/New_patches/2019-05-03-deprecate_useprojects.sql [hunk 1]
```diff
@@ -6,10 +6,10 @@ SELECT 'loris'
 WHERE NOT EXISTS (SELECT * FROM Project);
 
 -- associate all subprojects to the loris project by default if project table was empty
-INSERT INTO project_rel (ProjectID,SubprojectID)
-SELECT ProjectID, SubprojectID 
-FROM Project JOIN subproject 
-WHERE Project.Name='loris';
+INSERT IGNORE INTO Project (ProjectID, Name)
```
Comment (ridz1208) [inline SQL/New_patches/2019-05-03-deprecate_useprojects.sql]: @PapillonMcGill you replaced the wrong query

Comment (PapillonMcGill) [reply SQL/New_patches/2019-05-03-deprecate_useprojects.sql]: fixed!

**Comment by PapillonMcGill** [comment]
@ridz1208  Could you re-review

File: SQL/New_patches/2019-05-03-deprecate_useprojects.sql [hunk 2]
```diff
@@ -2,8 +2,8 @@ DELETE cs, c FROM ConfigSettings cs JOIN Config c ON c.ConfigID=cs.ID WHERE cs.N
 
 -- if Project table is empty, add default 'loris' project
 INSERT INTO Project (Name)
-SELECT 'loris'
-WHERE NOT EXISTS (SELECT * FROM Project);
+(SELECT 'loris' FROM DUAL
```
Comment (ridz1208) [inline SQL/New_patches/2019-05-03-deprecate_useprojects.sql]: @PapillonMcGill what is DUAL ?

Comment (driusan) [reply SQL/New_patches/2019-05-03-deprecate_useprojects.sql]: it's a fake table with exactly one row (it originates from Oracle, but the SQL standard doesn't really have any useable equivalent for cases like this and MySQL and MariaDB seem to both support it)

Comment (ridz1208) [reply SQL/New_patches/2019-05-03-deprecate_useprojects.sql]: lol... learning stuff everyday

**Pull request merged**